SOOUM/SOOUM/DesignSystem/Components/SOMCardTableViewCell.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  SOMCardTableViewCell.swift
+//  SOOUM
+//
+//  Created by JDeoks on 9/12/24.
+//
+
+import RxSwift
+import SnapKit
+import Then
+import UIKit
+
```

Comment:
```suggestion
import UIKit

import RxSwift
import SnapKit
import Then
```
서드파티 프레임워크는 퍼스트파티 프레임워크 뒤쪽으로 배치하는 게 좋을 거 같습니다.